### PR TITLE
re #17870 - Add "Catalan" to list of education languages on

### PR DESCRIPTION
**Pull request by defdf**
-Add Catalan to the .NET SDKs

### CODE REVIEW

File: schemas/2.0/language.xsd [hunk 1]
```diff
@@ -57,6 +57,7 @@
 	  <xs:enumeration value="Turkish" />
 	  <xs:enumeration value="Vietnamese" />	  
 	  <xs:enumeration value="Georgian" />
+      <xs:enumeration value="Catalan" />
```
Comment (Kralizek) [inline schemas/2.0/language.xsd:60]: Respect indentation

File: schemas/3.0/language.xsd [hunk 2]
```diff
@@ -57,6 +57,7 @@
 	  <xs:enumeration value="Turkish" />
 	  <xs:enumeration value="Vietnamese" />	  
 	  <xs:enumeration value="Georgian" />
+      <xs:enumeration value="Catalan" />
```
Comment (Kralizek) [inline schemas/3.0/language.xsd:60]: Respect indentation

**Review by Kralizek — CHANGES_REQUESTED** [review]

File: schemas/2.0/language.xsd (hunk 1 above)
Comment (defdf) [reply schemas/2.0/language.xsd:60]: I am aligning with the previous lines as the lines before Catalan is not.
image

File: schemas/3.0/language.xsd (hunk 2 above)
Comment (defdf) [reply schemas/3.0/language.xsd:60]: I am aligning with the previous line as the the lines befor 'Catalan' is not.
image

File: schemas/2.0/language.xsd (hunk 1 above)
Comment (Kralizek) [reply schemas/2.0/language.xsd:60]: my bad sorry. Can you align all of them? :)

**Review by Kralizek — APPROVED** [review]

**Pull request merged**